x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/common/percentiles_agg/config.ts

```diff
@@ -40,8 +40,36 @@ function parsePercentsInput(inputValue: string | undefined) {
 }
 
 // Input string should only include comma separated numbers
-function isValidPercentsInput(inputValue: string) {
-  return /^[0-9]+(,[0-9]+)*$/.test(inputValue);
+function validatePercentsInput(inputValue: unknown): {
```

Comment:
I noticed we don't have any unit tests for this file, would be great if you add some for this function and the updated check.